src/Data/Connection/MenuConnectionResolver.php

```diff
@@ -2,13 +2,26 @@
 
 namespace WPGraphQL\Data\Connection;
 
+use WPGraphQL\Model\Menu;
+
 /**
  * Class MenuConnectionResolver
  *
  * @package WPGraphQL\Data\Connection
  */
 class MenuConnectionResolver extends TermObjectConnectionResolver {
 
+	/**
+	 * @param $id
+	 *
+	 * @return mixed|null|\WPGraphQL\Model\Model|Menu
```

Comment:
According to the code, the correct output will be:

```suggestion
	 * @return Menu|null
```